src/test/java/org/folio/dataexp/service/TimerServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.dataexp.service;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.folio.dataexp.BaseDataExportInitializer;
+import org.folio.dataexp.client.OkapiClient;
+import org.folio.dataexp.domain.dto.RoutingEntry;
+import org.folio.dataexp.domain.dto.TimerDescriptor;
+import org.folio.spring.scope.FolioExecutionContextSetter;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
+
+import java.net.URI;
+import java.util.Collections;
+
+@TestPropertySource(properties = "application.clean-up-files-delay=10")
+class TimerServiceTest extends BaseDataExportInitializer {
```

Comment:
Hi @siarhei-charniak , seems the test could be simplify by using @ExtendWith(MockitoExtension.class) as there is no logic in test that need spring context usage. Could you please double check?